backend/schema/jsonvalidate.go

```diff
@@ -211,6 +216,7 @@ func ValidateJSONValue(fieldType Type, path path, value any, sch *Schema) error
 	return nil
 }
 
+// ValidateRequestMap validates a given JSON map against the provided schema.
 func ValidateRequestMap(ref *Ref, path path, request map[string]any, sch *Schema) error {
```

Comment:
Note for future reference: 
Since this function is only called by `ingress.BuildRequestBody` and `ValidateJSONValue` above, we could have this function call `TransformFromAliasedFields` directly. Then, the caller doesn't need to know to do it, so we can prevent issues like this in the future.

Discussed on Slack - we'll leave this as is for now to keep everything as stable as possible